src/meetings/components/sidebar/recordingAccordion/RecordingTimer.test.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import React from 'react';
+
+import { screen } from '@testing-library/react';
+
+import RecordingTimer from './RecordingTimer';
+import { setup } from '../../../../tests/test-utils';
+import { dateToISODate } from '../../../../utils/dateUtils';
+
+describe('RecordingTimer tests', () => {
+	test('Recording starts now', () => {
+		const startTime = dateToISODate(Date.now());
+		setup(<RecordingTimer timestamp={startTime} />);
+		expect(screen.getByText('00:00')).toBeVisible();
+	});
+
+	test('Recording starts 1 second ago', () => {
+		const startTime = dateToISODate(Date.now() - 1 * 1000);
+		setup(<RecordingTimer timestamp={startTime} />);
+		expect(screen.getByText('00:01')).toBeVisible();
+	});
+
+	test('Recording starts 1 minute ago', () => {
+		const startTime = dateToISODate(Date.now() - 1 * 60 * 1000);
+		setup(<RecordingTimer timestamp={startTime} />);
```

Comment:
If we export as a default value the timer formatted from the hook we can test these cases inside the hook test file